bodhi/server/consumers/__init__.py

```diff
@@ -15,4 +15,48 @@
 # You should have received a copy of the GNU General Public License
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-"""Contains Bodhi's fedmsg consumers."""
+"""
+fedora-messaging consumer.
+
+This module is responsible for consuming the messaging from the fedora-messaging bus.
+It has the role to inspect the topics of the message and call the correct handler.
+"""
+import logging
+
+import fedora_messaging
+
+
+from bodhi.server.consumers.composer import ComposerHandler
+from bodhi.server.consumers.signed import SignedHandler
+from bodhi.server.consumers.updates import UpdatesHandler
+
+
+log = logging.getLogger('bodhi')
+
+
+def messaging_callback(msg: fedora_messaging.api.Message):  # noqa: D401
+    """
+    Callback method called by fedora-messaging consume.
+
+    Redirect to messages to the correct handler in function of the
+    message topic.
+    """
+    log.info(
+        'Received message from fedora-messaging with topic: %s', msg.topic
```

Comment:
f-strings are faster and easier to read, and we can finally use them ☺

```suggestion
        f'Received message from fedora-messaging with topic: {msg.topic}'
```

Reply:
Cool I kept consistency with the old stuff. But more than happy to use f-strings \o/